js/server/modules/@arangodb/test-helper.js

```diff
@@ -195,5 +195,21 @@ exports.getDBServers = function() {
   return global.ArangoClusterInfo.getDBServers().map(x => ({id: x.serverId}));
 };
 
+exports.triggerMetrics = function () {
+  request({
+    method: "get",
+    url: "/_db/_system/_admin/metrics?mode=write_global",
+    headers: {accept: "application/json"},
+    body: {}
+  });
+  request({
+    method: "get",
+    url: "/_db/_system/_admin/metrics?mode=trigger_global",
+    headers: {accept: "application/json"},
+    body: {}
+  });
+  require("internal").sleep(2);
```

Comment:
same here. Please add a comment why we need this sleep and why 2 seconds?

Reply:
This code was written by @MBkkt. I just put implementation for server and for client in corresponding files.

Reply:
We need to wait a little bit unless we can be able to obtain metrics about inverted indexes on cluster deployment

Reply:
Some looping with huge overall timeout like 100s but check each 2s if metrics have arrived?  By this we will have realiable test in case of slow CI

Reply:
It is impossible to check for metrics right in `triggerMetrics` function because we don't know what and where to check values.
However, we can add this loop right in metrics value validation. 

For example something like this:
`
triggerMetrics();
// check link stats
{
    var isMetricsArrived = false;
    var timeToWait = 100;
    while (!isMetricsArrived) {
        let figures = db.TestCollection.getIndexes(true, true)
            .find(e => e.type === "arangosearch")
            .figures;
        try {
            assertNotEqual(null, figures);
            assertTrue(Object === figures.constructor);
            assertEqual(5, Object.keys(figures).length);
            assertEqual(0, figures.indexSize);
            assertEqual(0, figures.numDocs);
            assertEqual(0, figures.numLiveDocs);
            assertEqual(1, figures.numFiles);
            assertEqual(0, figures.numSegments);
            isMetricsArrived = true;
        } catch {
            isMetricsArrived = false;
            if (timeToWait > 0) {
                require("internal").sleep(2);
                timeToWait -= 2;
            } else {
                assertTrue(false);
            }
        }
    }
}
`
Is it OK?

Reply:
Yes, something like that